modules/nodes-cluster-resource-configure-jdk.adoc

```diff
@@ -29,20 +29,21 @@ this documentation, and may involve setting multiple additional JVM options.
 == Understanding how to override the JVM maximum heap size
 
 For many Java workloads, the JVM heap is the largest single consumer of memory.
-Currently, the OpenJDK defaults to allowing up to 1/4 (1/`-XX:MaxRAMFraction`)
-of the compute node's memory to be used for the heap, regardless of whether the
-OpenJDK is running in a container or not. It is therefore *essential* to
+
+Currently, on bare metal or VM setups, the OpenJDK's default heap value is 25%, 1/4th of the value (1/`-XX:MaxRamPercentage`). However, within containers using cgroups detection (v1 and v2 for JDK 11/17), the Xmx setting uses 50% of the maximum memory limits. In ubi9 - RHEL 9 images, the default allocation is 80%, leaving 20% for native use. It is therefore *essential* to
 override this behavior, especially if a container memory limit is also set.
 
+The `UseContainerSupport` flag is on by default in the JVM. It helps the JVM recognize Docker container operations, automatically fetching container-specific details like CPU count and total memory. Therefore, it is not necessary to manually define `JAVA_MAX_INITIAL_MEM`, as the JVM obtains maximum memory allocation from the pods directly.
+
 There are at least two ways the above can be achieved:
 
-* If the container memory limit is set and the experimental options are
-   supported by the JVM, set `-XX:+UnlockExperimentalVMOptions
-   -XX:+UseCGroupMemoryLimitForHeap`.
+* If the container memory limit is set and the default options are
+   supported by the JVM, set `-XX:
+   -XX:+UseContainerSupport`.
```

Comment:
line 41 and 42 should be: supported by the JVM, set `-XX:+UseContainerSupport`. (the original text has an additional -XX)

Reply:
Ack.